### DISCUSSION TITLE
How can I get 2019 directly

### DISCUSSION THREAD

**Discussion by Flutemr** [apache/dolphinscheduler#10973] [Q&A]
I want 2019, so I use $[yyyy-4], but it is 2022 after the test。
How can I get 2019 directly。thx。
![1657790056(1)](https://user-images.githubusercontent.com/30035663/178948988-1d78cda0-d6c3-4220-b9ac-6e9758e18032.jpg)
![1657790139(1)](https://user-images.githubusercontent.com/30035663/178949001-500eafa7-4da3-4eef-b7f4-ad136ae14043.jpg)

**Comment by Tandoy** [comment]
You can set it like this：$[add_months(yyyy,-12*3)]
![Snipaste_2022-07-15_20-44-00](https://user-images.githubusercontent.com/56899730/179225402-1dc294fe-ed2e-410e-a996-0ec77df62890.PNG)
